### PR TITLE
Migrate to single Dockerfile for ci-r

### PR DESCRIPTION
**Pull request by Shelnutt2**
Migrate to single Dockerfile for ci-r

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,9 +34,25 @@ To run:
...
+
+### Instructions
+
+1. Install the Docker daemon from https://www.docker.com/community-edition
+
+2. Clone the TileDB-Docker repo and build the images:
+```
+git clone https://github.com/TileDB-Inc/TileDB-Docker
+cd TileDB-Docker
+docker build -t tiledb:ci-r ci-r
```
Comment (eddelbuettel) [inline README.md]: If we build the container at hub.docker.com then pulling is easier than building so we could mention it.  But not that important.

File: ci-r/Dockerfile [hunk 2]
```diff
@@ -39,6 +39,23 @@ RUN apt-get update \
...
+    && mkdir build \
+    && cd build \
+    && cmake .. -DTILEDB_S3=ON -DTILEDB_HDFS=OFF -DCMAKE_BUILD_TYPE=DEBUG -DTILEDB_SERIALIZATION=ON \
+    && make -j$(nproc) \
+    && make install-tiledb \
+    && rm -rf /home/tiledb/TileDB-${version}
+
+RUN echo "/usr/local/lib" >> /etc/ld.so.conf.d/usr-local.conf
+RUN echo "/usr/local/lib64" >> /etc/ld.so.conf.d/usr-local.conf
+RUN ldconfig /usr/local/lib
```
Comment (eddelbuettel) [inline ci-r/Dockerfile]: That should probably be one `RUN` command, and the `ld.so.conf.d` entry is not needed on Ubuntu.

File: README.md (hunk 1 above)
Comment (Shelnutt2) [reply README.md]: The README is build instructions for in general.

File: ci-r/Dockerfile (hunk 2 above)
Comment (Shelnutt2) [reply ci-r/Dockerfile]: Removed/simplified

File: README.md (hunk 1 above)
Comment (Shelnutt2) [reply README.md]: For the CI build/deploy side we will eventually move this repo from travis to azure and setup azure for deploying to dockerhub. The end goal being to simplify and standardize our CI infrastructure to just Azure for our public repos.

**Pull request merged**